CHANGELOG.md

```diff
@@ -7,6 +7,10 @@
 
 # Release notes
 
+## 1.1.0
+
+This release adds a border on the unbranded app to differentiate it from a production app since the unbrandeis dsometimes use for training purposes [#278](https://github.com/medic/cht-android/issues/278).
```

Comment:
```suggestion
This release adds a border on the unbranded app to differentiate it from a production app since the unbranded app is sometimes used for training purposes [#278](https://github.com/medic/cht-android/issues/278).
```